web_external/js/models/SourceModel.js

```diff
@@ -0,0 +1,2 @@
+minerva.models.SourceModel = minerva.models.MinervaModel.extend({
```

Comment:
@mgrauer I agree but I would suggest let me push another brach for clean up and all. I wanted to get the paver working as well (which let you setup your own dev environment). In the next branch I will do all the clean you need. For now, I wanted the vagrant route to work. If you don't want to merge this into master that's fine since this does not branch any dev-code changes.